libtcmu_log.c

```diff
@@ -506,16 +506,16 @@ int tcmu_setup_log(void)
 	if (ret < 0)
 		tcmu_err("create stdout output error \n");
```

Comment:
This will be fixed in:

https://github.com/open-iscsi/tcmu-runner/pull/274/commits/10686538cbc7bfd557c2fd53f395bca596b164f4